### ISSUE TITLE
[grunt] watch task is misconfigured for sass and coffeescript

### ISSUE THREAD

**Issue by egladman** [CWDG/open-source-club-website#127]
watch task should include minifying the recompiled sass/coffeescript

**Comment by egladman** [comment]
i fixed this on branch: `search-improvements`, however I'm having trouble referencing the specific commit. Will close once `search-improvements` is merged into `master`

**Comment by smacz42** [comment]
#### WILL NOT FIX - new site
